sql/zone_settings.sql

```diff
@@ -22,10 +22,10 @@ CREATE TABLE `zone_settings` (
   `zoneip` tinytext NOT NULL,
   `zoneport` smallint(5) unsigned NOT NULL DEFAULT '0',
   `name` tinytext NOT NULL,
-  `music_day` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `music_night` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `battlesolo` tinyint(3) unsigned NOT NULL DEFAULT '0',
-  `battlemulti` tinyint(3) unsigned NOT NULL DEFAULT '0',
+  `music_day` smallint(3) unsigned NOT NULL DEFAULT '0',
```

Comment:
How can we migrate existing DB types? Is this assumed to fall on the server maintainers? Maybe a migration script is necessary?

Reply:
See migration 030 (home zone size) should be what you're looking for

Reply:
Done. Added migration and tested on a fresh db

Reply:
Zone_settings isn't protected and shouldn't need a migration.

Reply:
+1 to what coco said. It's a shame that zone_settings sits at the intersection of live/static data - but it is what it is

Reply:
Plus, if you have module'd inserts for this information, it's all forwards compatible anyway because the size of the field is going up